pkg/genericapiserver/options/etcd.go

```diff
@@ -83,4 +83,7 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&s.StorageConfig.Quorum, "etcd-quorum-read", s.StorageConfig.Quorum,
 		"If true, enable quorum read.")
+
+	fs.BoolVar(&s.StorageConfig.EnableCURLDebug, "enable-curl-debug", s.StorageConfig.EnableCURLDebug,
```

Comment:
I'm not sure if this debug info is available in the etcd server logs. But there is a variable "cURLDebug" in the curl.go which sets debug on, and nowhere to make this variable true.
If we do not want to expose this debug info, should we delete this variable as well?